CHANGELOG.md

```diff
@@ -64,12 +64,13 @@
   ```
 * Fixed copying of native structs for session errors and http requests. ([#924](https://github.com/realm/realm-dart/pull/924))
 * Fixed a crash when closing the SyncSession on App instance teardown. ([#5752](https://github.com/realm/realm-core/issues/5752))
+* Fixed sporadic generator generator failure. ([#879](https://github.com/realm/realm-dart/issues/879))
```

Comment:
@nielsenko Could you please change "Fixed sporadic generator generator failure." with "Fixed sporadic generator failure."